.github/workflows/tag-and-release.yaml

```
@@ -0,0 +1,19 @@
+    types: [ opened, unlabeled ]
+jobs:
+  permissions:
+    issues: write
+    pull-requests: write
+  label-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Test Local Action
+        uses: ./
```

actionlint

file "dist/index.js" does not exist in "/inmem/15/f4d46caf-997c-4cc6-a3ea-7bd8c4ce850d/home/jailuser/git". it is specified at "main" key in "runs" section in "Release Automation Action" action

(action)